packages/opencensus-core/src/stats/stats.ts

```diff
@@ -37,21 +42,23 @@ export class Stats {
   constructor(logger = defaultLogger) {
     this.logger = logger.logger();
 
-    // TODO (mayurkale): Decide how to inject MetricProducerForStats.
-    // It should be something like below, but looks like not the right place.
-
     // Create a new MetricProducerForStats and register it to
     // MetricProducerManager when Stats is initialized.
-    // const metricProducer: MetricProducer = new MetricProducerForStats(this);
-    // Metrics.getMetricProducerManager().add(metricProducer);
+    const metricProducer: MetricProducer = new MetricProducerForStats(this);
+    Metrics.getMetricProducerManager().add(metricProducer);
+  }
+
+  /** Gets the stats instance. */
+  static get instance(): Stats {
+    return this.singletonInstance || (this.singletonInstance = new this());
   }
 
   /**
    * Registers a view to listen to new measurements in its measure. Prefer using
    * the method createView() that creates an already registered view.
    * @param view The view to be registered
    */
-  registerView(view: View) {
+  registerView(view: View): void {
```

Comment:
Nit: do we need to include `: void` here? (Same comment applies to the other methods below)

Reply:
OK, I think that's fine.